packages/sdk-react/src/ui/CounterpartCell/CounterpartCell.tsx

```diff
@@ -19,18 +14,24 @@ interface Props {
 export const CounterpartCell = ({ counterpartId }: Props) => {
   const { data: counterpart, isLoading } = useCounterpartById(counterpartId);
 
-  const getCounterpartText = useCallback((counterpart: CounterpartResponse) => {
-    return counterpart.type === CounterpartType.ORGANIZATION
-      ? (counterpart as CounterpartOrganizationRootResponse).organization
-          .legal_name
-      : `${
-          (counterpart as CounterpartIndividualRootResponse).individual
-            .first_name
-        } ${
-          (counterpart as CounterpartIndividualRootResponse).individual
-            .last_name
-        }`;
-  }, []);
+  const getCounterpartText = useCallback(
+    (counterpart: components['schemas']['CounterpartResponse']) => {
+      return counterpart.type === 'organization'
```

Comment:
We have special functions, `isIndividualCounterpart` and `isOrganizationCounterpart,` which can calculate the correct type without using `as.`